apollo-federation/src/sources/connect/models/snapshots/validation_tests@invalid_entity_arg_on_field.graphql.snap

```diff
@@ -3,4 +3,4 @@ source: apollo-federation/src/sources/connect/models/validation.rs
 expression: "format!(\"{:?}\", errors)"
 input_file: apollo-federation/src/sources/connect/models/test_data/validation/invalid_entity_arg_on_field.graphql
 ---
-[Message { code: EntityNotOnRootQuery, message: "`@connect(entity:)` on `User.bestFriend` is invalid. Entities can only be declared on root `Query` fields.", locations: [Location { line: 15, column: 61 }..Location { line: 15, column: 73 }] }]
+[Message { code: EntityNotOnRootQuery, message: "`@connect(entity: true)` on `User.bestFriend` is invalid. Entities can only be declared on root `Query` fields.", locations: [Location { line: 15, column: 61 }..Location { line: 15, column: 73 }] }]
```

Comment:
```suggestion
[Message { code: EntityNotOnRootQuery, message: "`@connect(entity: true)` on `User.bestFriend` is invalid. Entity resolvers can only be declared on root `Query` fields.", locations: [Location { line: 15, column: 61 }..Location { line: 15, column: 73 }] }]
```

the entity is declared as `type User`, so i think it's more accurate to call this connector an "entity resolver".